### PR TITLE
Brokers: fix missing tooltip for config source

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
.
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@babel/core": "^7.16.0",
     "@babel/plugin-syntax-flow": "^7.18.6",
     "@babel/plugin-transform-react-jsx": "^7.18.6",
+    "@floating-ui/react": "^0.19.2",
     "@floating-ui/react-dom-interactions": "^0.10.3",
```
Comment (Mgrdich) [inline kafka-ui-react-app/package.json:11]: do we need this `@floating-ui/react-dom-interactions` when we are in the new version it is replaced with the `react` one.

Comment (David-DB88) [reply kafka-ui-react-app/package.json:11]: @floating-ui/react-dom-interactions  this not working form my task so i download @floating-ui/react

Comment (Mgrdich) [reply kafka-ui-react-app/package.json:11]: yeah i understood that point , what i am trying to say , if we no longer need `@floating-ui/react-dom-interactions` we should remove the lib.

please remove it and test and see if it works

Comment (David-DB88) [reply kafka-ui-react-app/package.json:11]: ок

Comment (David-DB88) [reply kafka-ui-react-app/package.json:11]: i removed  @floating-ui/react-dom-interactions, i checked now its works with  @floating-ui/react

**Review by Mgrdich — APPROVED** [review]

**Review by workshur — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**